src/lib.rs

```diff
@@ -92,6 +102,11 @@ pub fn hash(data: &[u8]) -> [u8; 32] {
     hashing_impl::sha3_256(data)
 }
 
+/// Generate a new unique nonce.
+pub fn gen_nonce() -> Nonce {
```

Comment:
what's the use of this - is it not misleading when you already have things like [these](https://github.com/maidsafe/safe_crypto/pull/15/files#diff-b4aea3e418ccdb71239b96952d9cddb6L288) ?